client/db/src/lib.rs

```diff
@@ -1124,6 +1124,14 @@ impl<Block: BlockT> Backend<Block> {
 		Self::from_database(db as Arc<_>, canonicalization_delay, &db_config, needs_init)
 	}
 
+	/// Reset the shared trie cache.
+	#[cfg(feature = "test-helpers")]
```

Comment:
```suggestion
```

We should not "hide" things behind special features. Because if the feature is enabled somewhere in the workspace, it gets enabled everywhere.